### PR TITLE
XSS js-year-calendar

### PR DESCRIPTION
**Pull request by bobbiejo7**
## ✍️ Description

 js-year-calendar is an open source calendar widget. XSS can be executed using the Name input area in the Event form section.

## 🕵️‍♂️ Proof of Concept

Go to https://year-calendar.github.io/js-year-calendar/examples scroll to Complete example, tap any date and the Event form will appear. Enter <script>alert('XSS')</script> in the text box labeled Name. Click Save. Tap the date where the XSS "event" was entered an alert will appear.

## 💥 Impact

Hijack an account
Spread web worms
Access browser history

## ☎️ Contact

no

## ✅ Checklist

**In my pull request, I have:**

- [x ] _Created and populated the `README.md` and `vulnerability.json` files_
- x[ ] _Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))_
- [x ] _Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))_
- [x ] _Proposed the CVSS vector items i.e. User Interaction, Attack Complexity_
- [x ] _Checked that the vulnerability affects the latest version of the package released_
- [x ] _Checked that a fix does not currently exist that remediates this vulnerability_
- [x ] _Complied with all applicable laws_

### CODE REVIEW

**Comment by toufik-airane** [comment]
To illustrate the vulnerability, @bobbiejo7 sent me screenshots in a Discord direct message.

Screenshot_20201216-122908_Samsung_Internet
Screenshot_20201216-122918_Samsung_Internet

**Comment by JamieSlome** [comment]
@bobbiejo7 - can you correct the `Repository` owner?

Cheers! 🍰

**Comment by bobbiejo7** [comment]
Hi. This is my first report for yall. What needs changed? To my knowledge
the repo owner is listed correctly?

File: bounties/npm/year-calendar/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "6.3"
+  },
+  "CVEs": [""], 
+  "Repository": {
+    "URL": "https://github.com/year-calendar/js-year-calendar",
+    "Codebase": ["JS"],
+    "Owner": "Paul-DS",
```
Comment (JamieSlome) [inline bounties/npm/year-calendar/1/vulnerability.json:40]: @bobbiejo7 - change this to `year-calendar`.

Cheers! 🍰

**Comment by JamieSlome** [comment]
@bobbiejo7 - closing this for now as it is going stale. Let me know if you have any questions or need any help and we can re-open.

Cheers! 🍰

**Pull request closed without merging**